worker.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+const amqp = require('amqplib');
+
+require('./models/Notification');
```

Comment:
????

Reply:
Moved schema definition to other file. This import is needed to run the code and make model available on mongoose instance.